test/API/ZoombaAPITestsData.py

```diff
@@ -336,8 +343,12 @@
 not_match_date_2 = "Key(s) Did Not Match:" \
                    "\n------------------" \
                    "\nKey: AssignedDate" \
-                   "\nExpected: 2015-12-16T14:21:58Z" \
-                   "\nActual: 1234-34-45" \
+                   "\nActual Date Not Correct Format:" \
+                   "\nExpected Formats: %Y-%m-%dT%H:%M:%S" \
+                   "\n                  %Y-%m-%dT%H:%M:%SZ" \
+                   "\n                  %Y-%m-%dT%H:%M:%S.%f" \
+                   "\n                  %Y-%m-%dT%H:%M:%S.%fZ" \
+                   "\nDate: 1234-34-45" \
```

Comment:
Why do we have a hard coded example of a bad date?  Shouldn't we return their actual formatted date?

Reply:
This is the response we're expecting from the test case.